src/commands/dev.ts

```diff
@@ -297,33 +328,15 @@ export async function command(commandOptions: CommandOptions) {
           }
           return resolvedImport;
         }
-        let [missingPackageName, ...deepPackagePathParts] = spec.split('/');
-        if (missingPackageName.startsWith('@')) {
-          missingPackageName += '/' + deepPackagePathParts.shift();
-        }
-        const [depManifestLoc] = resolveDependencyManifest(missingPackageName, cwd);
+        const packageName = getPackageName(spec);
```

Comment:
nit: slight preference for `getPackageNameFromSpecifier` or `getPackageNameFromImport`, since the argument is just a string we lose some context of when you'd want to call this.